### PR TITLE
Remove authors.md ato contributors.md

### PR DESCRIPTION
**Pull request by zhexuany**
---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tidb/3465)

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -0,0 +1,122 @@
+8cbx <8cbx@users.noreply.github.com>
```
Comment (shenli) [inline CONTRIBUTORS]: Seems this is not a valid email.

**Comment by zhexuany** [comment]
This is old school. But providing email address can let people contact contributors much more easily than giving a link which just redirects to user GitHub page.

File: CONTRIBUTORS (hunk 1 above)
Comment (zhexuany) [inline CONTRIBUTORS]: Yes. I already try my best to filter out some invalid email address. I need user identify themselves.

**Comment by coocood** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM @shenli

**Comment by zhexuany** [comment]
@shenli PTAL.

**Comment by shenli** [comment]
LGTM

**Pull request merged**